cocos2d/core/CCNode.js

```diff
@@ -2933,6 +2934,19 @@ var Node = cc.Class({
     onRestore: CC_EDITOR && function () {
         this._onRestoreBase();
 
+        /*
+         * TODO: Refine this code after completing undo/redo 2.0.
+         * The node is destroy when entering prefab edit mode, 
+         * but it is still used when exiting prefab edit mode, 
+         * so the built-in properties of the node need to be restored.
+        */
```

Comment:
The node will be destroyed when deleting in the editor,
but it will be reserved and reused for undo.